configs/mainnet.yaml

```diff
@@ -135,9 +135,12 @@ MESSAGE_DOMAIN_VALID_SNAPPY: 0x01000000
 SUBNETS_PER_NODE: 2
 # 2**8 (= 64)
 ATTESTATION_SUBNET_COUNT: 64
-ATTESTATION_SUBNET_EXTRA_BITS: 0
-# ceillog2(ATTESTATION_SUBNET_COUNT) + ATTESTATION_SUBNET_EXTRA_BITS
-ATTESTATION_SUBNET_PREFIX_BITS: 6
+# The granularity of the network.
+NETWORK_SHARD_COUNT: 64
```

Comment:
I've set this to 64, because in DAS I believe we are talking about either 32 or 64 extra topics. If it divides into 64 and is not larger than 64 then we dont get modulo bias and the mapping from shard to DAS column can be straight forward. 

I think this number only becomes an issue if we want to introduce more than 64 topics and have individual granularity.